### PR TITLE
multi: avoid busy waiting cpuminer goroutines.

### PR DESCRIPTION
**Pull request by dnldd**
This fixes cases where goroutines of the cpu miner busy wait causing the miner to hangin the process. Some cleanup has also been done. The expected cpu miner hash rate has been lowered to speed up harness tests and  the introductory blog post for dcrpool has been removed.

The harness has been updated to specify an appdata path to avoid modifying existing simnet setup and the fix to [off-by one nonce handling issues](https://github.com/decred/dcrd/commit/adff6a0bac899ced670663f964d78e4306703dd8) discovered in dcrd have been ported over.

### CODE REVIEW

File: intro.md [hunk 1]
```diff
@@ -1,24 +0,0 @@
-# Introducing dcrpool.
```
Comment (davecgh) [inline intro.md]: Did you intend to delete this file?

File: cmd/miner/client.go [hunk 2]
```diff
@@ -105,17 +104,19 @@ func (m *Miner) subscribe() error {
 	return nil
 }
 
+// sleep pauses goroutine execution for the provided duration.
+func (m *Miner) sleep(duration time.Duration) {
```
Comment (davecgh) [inline cmd/miner/client.go]: It doesn't really matter much, but I don't see why this is needed.  Why not just call `time.Sleep` directly?

File: intro.md (hunk 1 above)
Comment (dnldd) [reply intro.md]: yes, it has served its purpose and is no longer needed.

File: cmd/miner/client.go (hunk 2 above)
Comment (dnldd) [reply cmd/miner/client.go]: updated.

File: harness.sh [hunk 3]
```diff
@@ -224,6 +224,7 @@ fi
 
 if [ "${NETWORK}" = "simnet" ]; then
 tmux send-keys "dcrd -C ${DCRD_CONF} \
+--appdata=${NODES_ROOT}/master \
```
Comment (jholdstock) [inline harness.sh]: Should probably set the log dir as well?

Comment (dnldd) [reply harness.sh]: The default log directory is derived of the appdata set.

**Pull request merged**